tests/stream/test_stream_smoke/0018_query_state5_substream.json

```diff
@@ -93,17 +93,17 @@
                     "statements": [
                         {"client":"python", "query_type": "table", "query":"drop stream if exists test19_state_stream5"},
                         {"client":"python", "query_type": "table", "exist":"test19_state_stream5", "exist_wait":2, "wait":1, "query":"create stream test19_state_stream5 (id string, location string, value float, timestamp datetime64(3) default now64(3))"},
-                        {"client":"python", "query_type": "stream", "query_id":"19102", "wait":1, "terminate":"manual", "query":"subscribe to select id, count(*) over (partition by id), min(value) over (partition by id), max(value) over (partition by id) from test19_state_stream5 settings checkpoint_interval=1"},
+                        {"client":"python", "query_type": "stream", "query_id":"19102", "wait":1, "terminate":"manual", "query":"subscribe to select id, count(*) over (partition by id), min(value) over (partition by id), max(value) over (partition by id) from test19_state_stream5 settings checkpoint_interval=6"},
```

Comment:
I think we will need fix the crash for frequent ckpt case or at least we will need open a bug for this if frequent ckpt cause crash and fix it in a separate PR.

Reply:
for now, the ci only fail in 4 tests( verify on my local, the value incorrect).
previous I indeed encounter a coredump.